### PR TITLE
Update index.js

### PR DESCRIPTION
**Pull request by KSSaiTeja**
updated index.js with some readabilty code

fixes: #8

### CODE REVIEW

**Review by Pratyay360 — APPROVED** [review]
thanks for contributing

File: index.js [hunk 1]
```diff
@@ -10021,7 +10021,7 @@ function sleep(ms) {
 	return new Promise((resolve) => setTimeout(resolve, ms));
 }
 async function newTab() {
-	let used = [];
+	const used = [];
```
Comment (Pratyay360) [inline index.js:10024]: const used = new Set();

**Comment by Pratyay360** [comment]
there was a minor bug I have fixed it #10

File: index.js (hunk 1 above)
Comment (KSSaiTeja) [reply index.js:10024]: ah.. i forgot to do that.. anyway thanks for reviwing..

**Comment by KSSaiTeja** [comment]
> there was a minor bug I have fixed it #10 

yeah seen it.. anyway thanks for merging the PR.  can i share it on my socials

**Comment by Pratyay360** [comment]
yes you can

**Pull request merged**